package.json

```diff
@@ -65,6 +65,7 @@
     "@trivago/prettier-plugin-sort-imports": "2.0.2",
     "@types/jest": "26.0.23",
     "@types/node": "15.6.1",
+    "@types/react-select": "4.0.15",
```

Comment:
Are we sure we want to have types with a different package version than the package itself

Reply:
Yes we don't have choice here if we want to have types for react-select